lib/ecto/migration.ex

```diff
@@ -658,4 +659,17 @@ defmodule Ecto.Migration do
   defp validate_type!(%Reference{} = reference) do
     reference
   end
+
+  def prefix(opts_prefix) when is_nil(opts_prefix) do
```

Comment:
I'm thinking it won't work in normal use either, since in the migration modules the use imports only public functions.  We would still need the prefix/1 when the create macro is expanded I think.